content/commands/ft.profile/index.md

```diff
@@ -51,7 +51,7 @@ syntax_str: <SEARCH | AGGREGATE> [LIMITED] QUERY query
 title: FT.PROFILE
 ---
 
-Apply [`FT.SEARCH`]({{< baseurl >}}/commands/ft.search/) or [`FT.AGGREGATE`]({{< baseurl >}}/commands/ft.aggregate/) command to collect performance details
+Apply [`FT.SEARCH`]({{< baseurl >}}/commands/ft.search/) or [`FT.AGGREGATE`]({{< baseurl >}}/commands/ft.aggregate/) command to collect performance details.
 
 [Examples](#examples)
```

Comment:
```suggestion
[Examples](#examples)
```
I know this was here before, but could we include the link to the examples like in [FT.CREATE](https://redis.io/docs/latest/commands/ft.create/) instead?

Reply:
Done. See 4th commit.